packages/expo-cli/src/commands/export.js

```diff
@@ -56,9 +69,88 @@ export async function action(projectDir: string, options: Options = {}) {
     log('Terminating server processes.');
     await Project.stopAsync(projectDir);
   }
+
+  // Merge src dirs/urls into a multimanifest if specified
+  const mergeSrcDirs = [];
+
+  // src urls were specified to merge in, so download and decompress them
+  if (options.mergeSrcUrl.length > 0) {
+    // delete .tmp if it exists and recreate it anew
+    const rimrafP = util.promisify(rimraf);
+    const tmpFolder = path.resolve(projectDir, path.join('.tmp'));
+    await rimrafP(tmpFolder); // rm -rf tmpFolder
```

Comment:
Lately I've been trying to get rid of the separate `rimraf` and `mkdirp` libraries and to just use `fs-extra` since it can also do a `rm -rf` and returns promises from all functions without having to promisify anything:
```suggestion
    await fs.remove(tmpFolder);
```